core/server/data/migrations/versions/3.38/04-populate-recipient-filter-column.js

```diff
@@ -0,0 +1,30 @@
+const {createTransactionalMigration} = require('../../utils');
+const logging = require('../../../../../shared/logging');
+
+module.exports = createTransactionalMigration(
+    async function up(connection) {
+        logging.info('Updating emails.recipient_filter values based on posts.visibility');
+
+        await connection('emails')
+            .join('posts', 'emails.post_id', 'posts.id')
+            .update('emails.recipient_filter', 'paid')
+            .where('posts.visibility', 'paid');
+
+        await connection('emails')
+            .join('posts', 'emails.post_id', 'posts.id')
+            .update('emails.recipient_filter', 'all')
+            .where('posts.visibility', 'members');
+
+        await connection('emails')
+            .join('posts', 'emails.post_id', 'posts.id')
+            .update('emails.recipient_filter', 'all')
+            .where('posts.visibility', 'public');
+    },
+
+    async function down(connection) {
+        logging.info('Updating emails.recipient_filter values to paid');
+        await connection('posts')
+            .update('email_recipient_filter', 'paid');
```

Comment:
Code + comment does different things. Should be `await connection('emails').update('recipient_filter', 'paid');`

Reply:
Fixed!